validator/remote/src/main/java/tech/pegasys/teku/validator/remote/eventsource/EventSourceBeaconChainEventAdapter.java

```diff
@@ -128,7 +133,16 @@ public void onPrimaryNodeBackReady() {
   }
 
   private BackgroundEventSource createEventSource(final RemoteValidatorApiChannel beaconNodeApi) {
-    final HttpUrl eventSourceUrl = createHeadEventSourceUrl(beaconNodeApi.getEndpoint());
+
+    final List<EventType> eventTypes = new ArrayList<>();
+    eventTypes.add(EventType.head);
+    if (shutdownWhenValidatorSlashedEnabled) {
+      eventTypes.add(EventType.attester_slashing);
+      eventTypes.add(EventType.proposer_slashing);
+    }
```

Comment:
ideally test case here to so we don't regress...

Reply:
done